surround_cli/templates/new/web_runner.py.txt

```diff
@@ -21,45 +28,35 @@ class WebRunner(Runner):
     def run(self, mode=RunMode.PREDICT):
         # Setup web service
         self.assembler.init_assembler()
-        self.application = Application(self.assembler)
-        self.application.listen(8080)
-        logging.info("Server started at http://localhost:8080")
-        logging.info("Available endpoints:")
-        logging.info("* GET   /info                      # Version info")
-        logging.info("* POST  /estimate                  # Send JSON data to the Surround pipeline")
-        tornado.ioloop.IOLoop.instance().start()
+        HELPER.assembler = self.assembler
+        uvicorn.run(
+            APP, host="0.0.0.0", port=8080, log_level="info"
```

Comment:
Change to different default port to not conflict with Spring Boot for shared projects used at A2I2.